python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -239,15 +241,16 @@ class Dropout(HybridBlock):
         `Dropout: A Simple Way to Prevent Neural Networks from Overfitting
         <http://www.cs.toronto.edu/~rsalakhu/papers/srivastava14a.pdf>`_
     """
-    def __init__(self, rate, **kwargs):
+    def __init__(self, rate, axes=(), **kwargs):
```

Comment:
Default to None?

Reply:
This is the same default value as the one added in Dropout op. I set it this way so that I don't have to handle None.